### PR TITLE
[stdlib]Add an opaque flavor of the normalization tests

### PR DESCRIPTION
**Pull request by lancep**
Run the normalization tests through the opaque path as well.

### CODE REVIEW

File: validation-test/stdlib/StringNormalization.swift [hunk 1]
```diff
@@ -9,16 +9,28 @@
...
 //
 //===----------------------------------------------------------------------===//
-// RUN: mkdir -p %t
-// RUN: %target-build-swift %s -o %t/a.out
+// RUN: %empty-directory(%t)
+// RUN: if [ %target-runtime == "objc" ]; \
+// RUN: then \
+// RUN:   %target-clang -fobjc-arc %S/Inputs/NSSlowString/NSSlowString.m -c -o %t/NSSlowString.o && \
+// RUN:   %target-build-swift -I %S/Inputs/NSSlowString/ %t/NSSlowString.o %s -o %t/a.out; \
+// REQUIRES: objc_interop
```
Comment (jrose-apple) [inline validation-test/stdlib/StringNormalization.swift]: Did you mean to move this? Because moving it doesn't do anything.

**Review by milseman — APPROVED** [review]
LGTM!

**Comment by lancep** [comment]
@swift-ci please test

**Comment by lancep** [comment]
@swift-ci please test

**Comment by lancep** [comment]
@swift-ci please test

**Pull request merged**